compiler/rustc_infer/src/infer/opaque_types.rs

```diff
@@ -309,14 +309,21 @@ impl<'a, 'tcx> InferCtxt<'a, 'tcx> {
     /// the same as generating an outlives constraint on `Tc` itself.
     /// For example, if we had a function like this:
     ///
-    /// ```rust
+    /// ```ignore HELP (why doesn't this work?)
```

Comment:
https://github.com/rust-lang/rust/issues/85433
The workaround is to add `fn main() {}` to the top.
```suggestion
    /// ```
    /// # fn main() {}
```